src/ConfigScenes/ConfigScene.js

```diff
@@ -0,0 +1,79 @@
+// js/ConfigScene.js
+import {LANGUAGES, textStyle} from '../Config.js';
+
+export class ConfigScene extends Phaser.Scene {
+    constructor() {
+        super({ key: 'ConfigScene' });
+        this.language = 'EN'; // Default language
+    }
+
+    init(data) {
+        this.data = data;
+    }
+
+    create() {
+        this.initData()
+        this.initiateScreen();
+
+    }
+
+    initData(){
+        this.numberOfPanels = 2;
+        this.nextScene = 'LanguageScene';
+        this.options = [
+            {dummy: 'DUMMY1'},
+            {dummy: 'DUMMY2'}
+
+            // { language: 'sk' },
+            // { language: 'en' }
+        ]
+        this.optionTexts = [
+            "Dummy1",
+            "Dummy2"
+            // `${LANGUAGES.sk.language} (${LANGUAGES.sk.lang})`,
+            // `${LANGUAGES.en.language} (${LANGUAGES.en.lang})`
+        ]
+    }
+
+    initiateScreen() {
+        const graphics = this.add.graphics();
+        const width = this.cameras.main.width;
+        const height = this.cameras.main.height;
+
+        this.makePanels(this.numberOfPanels, graphics, width, height)
+
+        let panelWidth = width/this.numberOfPanels;
+
+        this.input.on('pointerdown', (pointer) => {
+            console.log(this.optionTexts[(pointer.x / panelWidth)|0]+" selected");
+            let newData = {...this.data,...this.options[(pointer.x / panelWidth)|0]}
+            this.scene.start(this.nextScene, { language: 'sk' });
+
+        });
+    }
+
+    color(count, index) {
+        if (count === 2) {
+            return index === 0 ? 0x00ff00 : 0xff0000;
```

Comment:
Farby by som dal ako konstanty aj tak sa tam opakuju, plati na viacerych miestach

Reply:
snažil som sa to spraviť tak že to bude pekne dynamické pre situáciu kedy by hypoteticky bolo treba viac možností (v spojitosti s tým že tu mala byť dedičnosť, ako som opisoval tam kde si mi vyčítal duplicitný kód) ale retrospektívne, hej, hento je zbytočné...